api/buildout.cfg

```diff
@@ -72,14 +83,15 @@ eggs = ${instance:eggs}
 [versions]
 setuptools =
 zc.buildout =
-plone.restapi = 8.16.2 # force to latest p.restapi
-plone.rest = 2.0.0a1 # new JSON only traversal
+plone.restapi = >=8.13.0 # force to latest p.restapi
```

Comment:
@rpatterson I'd prefer exact version pins here. CI would fail without notice when something changes in newer versions that we fetch.

Reply:
All the rest of the version overrides in that buildout configuration are explicitly *clearing* version pins.  There are 14 version overrides there, only 2 of them were pins before my changes and those 2 are probably out of date.  So the CI failure risk you talk about is already there in *most* of those version overrides and unrelated to the changes in this PR, no?

IMO, it's important to accomplish *both* goals in one way or another:
1. Version constraints that define the *requirements* (E.g, we need the feature in this library that was introduced in v1.1 -> `foo.bar >= 1.1`)
2. Version pins for reproducability

I'd be happy to submit a separate PR to deal with this separate issue if that's what others want but I don't think it should be stacked on top of this PR.

Reply:
[Submitted a separate PR](https://github.com/plone/volto/pull/2922#issue-804005730)